app/models/user.rb

```diff
@@ -399,21 +399,21 @@ def workshop_eligible?
   end
 
   def admin_organizations
-    org_ids = organization_memberships.where(type_of_user: "admin").pluck(:organization_id)
+    org_ids = organization_memberships.admin.pluck(:organization_id)
     organizations.where(id: org_ids)
   end
 
   def member_organizations
-    org_ids = organization_memberships.where(type_of_user: %w[admin member]).pluck(:organization_id)
+    org_ids = organization_memberships.member.pluck(:organization_id)
     organizations.where(id: org_ids)
   end
 
   def org_member?(organization)
-    OrganizationMembership.exists?(user: self, organization: organization, type_of_user: %w[admin member])
+    organization_memberships.member.exists(organization: organization)
```

Comment:
Think this is missing a question mark after `exists`

Reply:
I _also_ get confused moving between sql (where it's an `EXISTS` clause, what this query generates) and ruby (where it's `exists?`, following the pattern for predicates).

Reply:
@Zhao-Andy Yeah, good catch.  The tests grabbed it as well.